alignak/objects/contact.py

```diff
@@ -67,28 +68,28 @@ class Contact(Item):
     """Host class implements monitoring concepts for contact.
     For example it defines host_notification_period, service_notification_period etc.
     """
+    name_property = "contact_name"
     my_type = 'contact'
 
     properties = Item.properties.copy()
     properties.update({
         'contact_name':
             StringProp(fill_brok=['full_status']),
-        'alias':
-            StringProp(default='none', fill_brok=['full_status']),
         'contactgroups':
             ListProp(default=[], fill_brok=['full_status']),
+        # Todo: define default parameters for contact notifications
         'host_notifications_enabled':
             BoolProp(default=True, fill_brok=['full_status']),
         'service_notifications_enabled':
             BoolProp(default=True, fill_brok=['full_status']),
         'host_notification_period':
-            StringProp(default='', fill_brok=['full_status']),
+            StringProp(fill_brok=['full_status']),
         'service_notification_period':
-            StringProp(default='', fill_brok=['full_status']),
+            StringProp(fill_brok=['full_status']),
         'host_notification_options':
-            ListProp(default=[''], fill_brok=['full_status'], split_on_coma=True),
+            ListProp(default=[''], fill_brok=['full_status']),
```

Comment:
split_on_coma=True is the default value, why specifying it? Only defined split_on_coma when it is set to False ...